docs/api/ethereum.md

```diff
@@ -357,6 +358,7 @@ A `class` providing communication with the Ethereum blockchain through [Bitski](
 | options.mutationTimeout | A `number` representing the number of milliseconds in which a mutation times out. Defaults to `3600000`. You can set it to `-1` to disable timeout.
```

Comment:
| options.mutationTimeout | The `number` of milliseconds after which a mutation times out. Defaults to `3600000`. You can set it to `-1` to disable the timeout.